client/src/components/Items.js

```diff
@@ -1,5 +1,6 @@
 import React, {useEffect, useState} from 'react'
 import '../styles/styles.css'
+import '../styles/Item.css'
```

Comment:
do we really need it? 
let's use one file for styles, just add everything from Item.css there